cmd/ethrex/ethrex.rs

```diff
@@ -14,6 +14,8 @@ use tracing::info;
 
 #[cfg(any(feature = "l2", feature = "based"))]
 use ethrex::l2::L2Options;
+#[cfg(feature = "l2")]
```

Comment:
We'd want this also in `based`.
```suggestion
#[cfg(any(feature = "l2", feature = "based"))]
```

Reply:
but `based` should automatically enable `l2`, right?

Reply:
> but based should automatically enable l2, right?

yes: https://github.com/lambdaclass/ethrex/blob/ef87b0fee249cec506c63505f6b6b386770f69bb/cmd/ethrex/Cargo.toml#L74

Reply:
You are right